scaling_performance/cluster_limits.adoc

```diff
@@ -0,0 +1,102 @@
+[[scaling-performance-cluster-limits]]
+= Cluster Limits
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+This topic summarizes the limits for objects in {product-title}.
+
+In most cases, exceeding these thresholds results in lower overall performance.
+It does not necessarily mean that the cluster will fail over.
+
+Some of the limits represented in this topic are given for the largest possible
+cluster. For smaller clusters, the limits are proportionally lower.
+
+There are many factors that influence the stated thresholds, including the etcd
+version or storage data format.
+
+[[scaling-performance-current-cluster-limits]]
+== Current Cluster Limits
+
+[options="header",cols="3,2"]
+|===
+| Limit Type |Current Limit
+
+|Total number of all objects footnoteref:[allobjects,There is a limitation on the total number of objects on the system, as this affects etcd and its resource consumption.]
+| 250,000
+
+| Number of nodes footnoteref:[numberofnodes,Having clusters with more than 5000 nodes is not the best option. Consider splitting into multiple clusters.]
+| 2,000
+
+| Number of pods
+| 120,000
+
+| Number of pods per node footnoteref:[podspernodeorcore, The limit for number of pods on a given node is the minimum from the _pod per node_ and _pods per core multiplied by the number of cores of a node_.]
+| 250
+
+| Number of pods per core footnoteref:[podspernodeorcore]
+| 10
```

Comment:
This is tricky...10 is the default, but it's not the supported maximum.  People can turn it off and still be supported.
It is however an out of the box default that people do hit now and then, so I don't think it should be left off this limits page, just that somehow we need to indicate that it is not the maximum supported number.  The maximum supported will always be pods-per-node.